### PR TITLE
Feature/use astropy units

### PR DESCRIPTION
**Pull request by captain-exoplanet**
This PR includes: 

- Using astropy.units to keep track of the velocities within rv.py and eliminate any unit conversions issues.
- Mirek's modifications to the wavelength solution from the feature/improved _wavelength_solution branch.

### CODE REVIEW

File: ExampleReduction.ipynb [hunk 1]
```diff
@@ -216,6 +216,23 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "instrument = dbs.get_instruments_at_site('lsc', settings.db_address)[0]"
+   ]
+  },
```
Comment (gmbrandt) [inline ExampleReduction.ipynb:235]: lets do a 'git checkout dev banzai_nres/ExampleReduction.ipynb' to clear these changes

File: banzai_nres/rv.py [hunk 2]
```diff
@@ -49,7 +46,7 @@ def cross_correlate(velocities, wavelength, flux, flux_uncertainty, template_wav
     variance = flux_uncertainty * flux_uncertainty
 
     for v in velocities:
-        doppler = 1.0 / (1.0 + v / c)
+        doppler = 1.0 * units.dimensionless_unscaled / (1.0 * units.dimensionless_unscaled + v / constants.c)
```
Comment (gmbrandt) [inline banzai_nres/rv.py:49]: I have had issues in the past with astropy units massively slowing down computation time in loops. Just want to verify that this does not slow down the RV computation.

**Review by gmbrandt — APPROVED** [review]
Approve assuming the tests pass.

File: ExampleReduction.ipynb (hunk 1 above)
Comment (captain-exoplanet) [reply ExampleReduction.ipynb]: Done.

**Review by cmccully — APPROVED** [review]

**Pull request merged**